src/scripts/clipperUI/components/previewViewer/previewViewerAugmentationHeader.tsx

```diff
@@ -52,23 +52,25 @@ class PreviewViewerAugmentationHeaderClass extends PreviewViewerHeaderComponentB
 	private getSerifGroup(): ControlGroup {
 		return {
 			id: Constants.Ids.serifControl,
+			role: "radiogroup",
+			isAriaSet: true,
 			innerElements: [
 				<button
 					aria-label={Localization.getLocalizedString("WebClipper.Accessibility.ScreenReader.ChangeFontToSansSerif")}
-					aria-pressed={!this.props.serif ? "true" : "false"}
+					aria-checked={!this.props.serif ? "true" : "false"}
 					id={Constants.Ids.sansSerif}
-					{...this.enableInvoke(this.props.changeFontFamily, 101, false) }
+					{...this.enableInvoke(this.props.changeFontFamily, !this.props.serif ? 101 : undefined, false, undefined, Constants.AriaSet.serifGroupSet) }
 					className={!this.props.serif ? HeaderClasses.Button.activeControlButton : HeaderClasses.Button.controlButton}
-					type="button">
+					role="radio">
```

Comment:
Rather than change the role to radio, we should change this element from a button to an input element of type="radio".